nifi-nar-bundles/nifi-cassandra-bundle/nifi-cassandra-processors/src/main/java/org/apache/nifi/processors/cassandra/PutCassandraRecord.java

```diff
@@ -57,6 +60,11 @@
 @CapabilityDescription("This is a record aware processor that reads the content of the incoming FlowFile as individual records using the " +
         "configured 'Record Reader' and writes them to Apache Cassandra using native protocol version 3 or higher.")
 public class PutCassandraRecord extends AbstractCassandraProcessor {
+    static final String UPDATE_TYPE = "UPDATE";
```

Comment:
These should probably be `AllowableValue` instances rather than `String`, that way you can still include the name, but also a display name and a description (which is used for tooltips). You can still add them the same way to the property (since the `.allowableValues()` method accepts either Strings or AllowableValues.